pkg/trace/stats/concentrator_test.go

```diff
@@ -41,10 +45,21 @@ func testSpan(spanID uint64, parentID uint64, duration, offset int64, service, r
 		Duration: duration,
 		Start:    getTsInBucket(alignedNow, testBucketInterval, offset) - duration,
 		Service:  service,
-		Name:     "query",
+		Name:     name,
 		Resource: resource,
 		Error:    err,
 		Type:     "db",
+		Meta:     meta,
+	}
+}
+
+// countsEq is a test utility function to assert expected == actual for count aggregations.
+func countsEq(assert *assert.Assertions, expected map[string]float64, actual map[string]Count) {
```

Comment:
Encapsulated this logic to help simplify some test code. There are probably other simplifications that could be made but I didn't want to get carried away.

Reply:
That's fine, but a few things:
* Can it take param `*testing.T` even if it uses `assert` inside?
* We need a `TestCountsEq`. Test helpers need to be tested too 🙂